custom_components/garmin_connect/sensor.py

```
@@ -148,178 +130,89 @@ async def async_setup_entry(
         },
         "add_blood_pressure",
     )
 
 
-class GarminConnectSensor(CoordinatorEntity, SensorEntity):
+class GarminConnectSensor(GarminConnectEntity, SensorEntity):
     """Representation of a Garmin Connect Sensor."""
 
-    _attr_has_entity_name = True
-
     def __init__(
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)